web/packages/teleterm/src/ui/services/workspacesService/workspacesService.ts

```diff
@@ -72,25 +75,15 @@ export interface WorkspacesState {
    * of the app. It is useful in places that want to wait for the state to be restored before
    * proceeding.
    *
-   * If during the previous start of the app the user was logged into a workspace which cert has
-   * since expired, isInitialized will be set to true only _after_ the user logs in to that
-   * workspace (or closes the login modal).
-   *
    * This field is not persisted to disk.
-   *
-   * Side note: Arguably, depending on the use case, the moment isInitialized is set to true could
-   * be changed to happen right before the modal is shown. Ultimately, the thing that interests us
-   * the most is whether the state from disk was loaded into memory. Maybe in the future we will
-   * need to separate values or an enum.
-   *
    */
   isInitialized: boolean;
```

Comment:
As discussed on a call, rename to something like `hasPulledInitialState`, move setting this field out of `restorePersistedState` into a separate method that's called from `pullInitialState`.